ocean_provider/file_types/file_types_factory.py

```diff
@@ -59,8 +59,13 @@ def validate_and_create(file_obj) -> Tuple[bool, Any]:
                     userdata=file_obj.get("userdata"),
                 )
             else:
+                logger.debug(f"Unsupported type {file_obj}")
```

Comment:
shouldn't it be file_obj["type"]? I think another approach to this (and the other addition you made in this file) would be to log the message upon return, if the first member of the tuple is False. Otherwise, please assign the message variable first and use it in both debugging and the return statement.